activesupport/test/abstract_unit.rb

```diff
@@ -40,3 +40,13 @@ def jruby_skip(message = "")
       skip message if defined?(JRUBY_VERSION)
     end
 end
+
+if ENV["CI"]
+  require "minitest/reporters"
+  Minitest.singleton_class.prepend(Module.new do
+    def plugin_rails_init(*)
+      super
+      reporter << Minitest::Reporters::JUnitReporter.new
```

Comment:
@y-yagi

Is this implementation acceptable?
If it's ok, I'll add this to every gem as well.

Reply:
Hmmm. Just tricky to me.How about treating it as a minitest plugin? 

```diff
--- a/activesupport/test/abstract_unit.rb
+++ b/activesupport/test/abstract_unit.rb
@@ -40,3 +40,14 @@ def jruby_skip(message = "")
       skip message if defined?(JRUBY_VERSION)
     end
 end
+
+if ENV["CI"]
+  module Minitest
+    def self.plugin_rails_ci_init(*)
+      require "minitest/reporters"
+      reporter << Minitest::Reporters::JUnitReporter.new
+    end
+  end
+
+  Minitest.extensions.unshift "rails_ci"  
+end
```